tests/test_tokenization_character_bert.py

```diff
@@ -0,0 +1,211 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import unittest
+
+from transformers.models.character_bert.tokenization_character_bert import (
+    BasicTokenizer,
+    CharacterBertTokenizer,
+    CharacterMapper,
+    _is_control,
+    _is_punctuation,
+    _is_whitespace,
+)
+from transformers.testing_utils import require_tokenizers
+
+from .test_tokenization_common import filter_non_english
+
+
+@require_tokenizers
+class BertTokenizationTest(unittest.TestCase):
```

Comment:
This should be a `CharacterBertTokenizationTest`, and it should either depend from the `TokenizerTesterMixin` or override more of its tests. Currently, untested behavior includes padding, overflowing tokens, truncating, saving and loading, etc.

If you have found this too hard to do as your tokenizer is very peculiar, let me know and we'll be happy to take over that last step!

Reply:
Changed the name, thanks :).

I'll give the tests a shot and I'll let you know if I have any issues.

Reply:
Okay, it's done @LysandreJik. I think that everything is good now 😊